c_common/models/system_models/src/extra_monitor_support.c

```diff
@@ -157,15 +157,21 @@ enum {
     RTR_BLOCKED_BIT = 25,
     RTR_FPE_BIT = 17,
```

Comment:
Seems that RTR_FPE_BIT  should be 18, as the number of processor cores.

Reply:
Oops - fixed it now!

Reply:
Re-fixed this. (It's buggy on master.)